lab_3_generate_by_ngrams/main.py

```diff
@@ -270,6 +369,22 @@ def run(self, seq_len: int, prompt: str) -> Optional[str]:
         In case of corrupt input arguments or methods used return None,
         None is returned
         """
+        if not isinstance(seq_len, int) or not isinstance(prompt, str) or len(prompt) == 0:
+            return None
+        encoded = self._text_processor.encode(prompt)
+        n_gram_size = self._model.get_n_gram_size()
+        if not (encoded and n_gram_size):
+            return None
+        for i in range(seq_len):
+            next_tokens = self._model.generate_next_token(encoded[-(n_gram_size + 1):])
+            if not next_tokens:
+                break
+            best_next = [token for token, freq in next_tokens.items() if freq == max(next_tokens.values())]
```

Comment:
sorted by freq and get first - it is more clear

Reply:
I changed it, so fixed?